plugins/chat/spec/lib/guardian_extensions_spec.rb

```diff
@@ -299,8 +299,18 @@
       end
 
       context "when category has a channel" do
-        it "does not allow to delete the category" do
-          expect(staff_guardian).not_to be_able_to_delete_category(category)
+        context "when channel has no messages" do
+          it "allows to delete the category" do
```

Comment:
should there be another assertion here for something like:

```
it "deletes the channels that have no messages" do...
```

Reply:
Agreed we might want higher level test here, not in this file, but some test where category.destroy is actually called

Reply:
Going to add a request spec for this then.